### PR TITLE
Added mxgraph type dropdown

### PR DESCRIPTION
**Pull request by Daedo**

### CODE REVIEW

**Review by wehrlean — CHANGES_REQUESTED** [review]
I am sorry for the work you spent in this feature, but it I think a more consequent way would be to use the popups implemented by @sebeder in PR #26 and add there a new item to change the type of the node. 
Your solution is more visible, but I think the popups are the better way. Should be discussed in the JF

**Comment by Daedo** [comment]
After having discussed it we want to add it, since it is more visible.

File: web/src/app/modules/views/main/editors/modules/graphical-editor/providers/properties/vertex-provider.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { IContainer } from '../../../../../../../../model/IContainer';
 import { mxgraph } from 'mxgraph'; // Typings only - no code!
 import { IModelNode } from '../../../../../../../../model/IModelNode';
 import { ShapeProvider} from './shape-provider';
-import {ValuePair} from './value-pair';
+import {CEGmxModelNode} from './ceg-mx-model-node';
 import { ConverterBase } from '../../converters/converter-base';
 import {VariableAndCondition} from '../../converters/variable-condition-name-converter';
```
Comment (wehrlean) [inline web/src/app/modules/views/main/editors/modules/graphical-editor/providers/properties/vertex-provider.ts:8]: Remove unused imports

**Review by wehrlean — CHANGES_REQUESTED** [review]
This way is definitive more visible. Two bugs:

- If you set the type to _OR_, save it and reload the page, the dropdown is set to _AND_ again
- Autolayout leads to an error, because the type is empty
- Remove tooltip for the dropdown (see trello)
- Comment

**Review by wehrlean — APPROVED** [review]
Good, all things work!

**Pull request merged**